src/Http/SecurityMiddleware.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Valentin\Mojam\Http;
+namespace ShaitanMasters\Prometheus\Http;
 
 use Closure;
 use Illuminate\Http\Response;
```

Comment:
lets check HEADER for security **KEY**
key name should be configured in config file